.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
We should probably add `.vscode` to `.gitignore`

Reply:
i will remove my new additions and add it to the gitignore - should we just remove the dir all together?

Reply:
i went ahead and just removed the entire dir